realsense2_camera/include/mavros_timestamp.h

```diff
@@ -0,0 +1,432 @@
+/*
```

Comment:
Are you trying to conform to the realsense style here? If not run the file through an autoformatter.

Reply:
got confused, mavros_timestamp should have been google style. anyway, run autoformatter